docs/admin/templates/extending-templates/parameters.md

```diff
@@ -313,6 +313,58 @@ data "coder_parameter" "project_id" {
 }
 ```
 
+## Workspace presets
+
+Workspace presets allow you to select settings tailored to your most common development scenarios.
+
+Coder admins can configure workspace presets with combinations of parameter values and workspace settings within each template's Terraform file, and workspace users can customize their configurations as they need to.
```

Comment:
What do you mean by "and workspace settings" here?

Reply:
"workspace users can customize their configurations as they need to" is not something that is possible right now. Users can only select from available presets, but they cannot customize these configurations.

Reply:
re "workspace settings" and customizing

I was working off the PR/FAQ:

> pre-selection of settings tailored to the most common development scenarios.
...
> allowing users to override settings when fine-tuning or custom configurations are required.

would it work better framed as parameters?

```diff
Workspace presets allow you to select
- settings tailored to your most
+ common groups of parameters tailored to your most
common development scenarios.
```

Reply:
Generally this feature can be described as:

"Workspace presets allow you to configure commonly used combinations of parameters into a single option, which makes it easier for developers to pick one that fits their needs". Need not be verbatim but just the general gist.